src/types.ts

```diff
@@ -45,3 +45,14 @@ export interface BrowserPageInterface {
 export type BrowserPageEvents = 'load' | 'close' | 'dialog' | 'response';
 export type Unsubscribe = () => void;
 export type StringifiedCookies = string;
+
+export type GetWinningNumbersResponseType = {
```

Comment:
```suggestion
export type GetWinningNumbersResponse = {
```

타입에는 타입을 빼자! (인터페이스도  빼도 되긴 하는데, implements 해서 구현해줘야 하는 애들은 postfix 로 Interface를 붙였음)